### PR TITLE
Added Image Slider

### PR DESCRIPTION
**Pull request by samarsault**
Added Image Slider to GCI Site
Task : http://www.google-melange.com/gci/task/view/google/gci2014/5551653621071872

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,229 +107,243 @@ <h1 class="section__title">Mentors</h1>
...
-                </div>
-                <div class="sympl onethird">
-                    <div class="box-team wow tada" data-wow-delay="0.1s">
-                        <section class="person person--mentor">
-                            <div class="person__img">
-                                <div class="person__img__info">
-                                    <div class="person__img__info__cont">
-                                        <a href="https://github.com/mariobehling" target="_blank"><i class="icon-github"></i></a>
-                                    </div>
+                                      -->
```
Comment (roonyh) [inline index.html]: Why this indentation change?

**Comment by roonyh** [comment]
Hi! This will be a nice addition to the site!! Thanks much!

But it seems that you are making some unnecessary indentation changes to the index.html file. Can you please remove them and commit only the necessary changes?

**Comment by samarsault** [comment]
I formatted the index.html file

File: index.html [hunk 2]
```diff
@@ -1,303 +1,342 @@
...
-
-                <div class="menu">
-                    <nav>
-                        <ul>
-                            <li><a href="http://blog.fossasia.org/">Blog</a>
-                            </li>
-                        </ul>
-                    </nav>
+    <!--<![endif]-->
+    <head>
```
Comment (roonyh) [inline index.html]: keep this aligned with `<html>` tag

**Comment by roonyh** [comment]
Now the index.html file shows changes in every line. see: https://github.com/fossasia/fossasia.github.io/pull/45/files

Its because you have indented `<head>` and `<body>` tags. Your change probably is correct but its a little difficult to see what you exactly changed to add the slider.

**Comment by samarsault** [comment]
Alright, I fixed the indentation. Is the new one ok ?

**Comment by roonyh** [comment]
Yep that works. Thanks. Will merge this soon.

**Comment by roonyh** [comment]
I tested this locally. It works fine! Can you tell me where you got the images from?

**Comment by samarsault** [comment]
I got the images from fossasia's flickr page provided on the task website

**Comment by roonyh** [comment]
Please "finish" the task

**Comment by roonyh** [comment]
[merged](https://github.com/fossasia/fossasia.github.io/commit/1c28e860bcb849b4ca43beeaf1e761c20904f329)

**Pull request closed without merging**